samza-core/src/main/java/org/apache/samza/config/StorageConfig.java

```diff
@@ -47,7 +47,7 @@
   public static final String ACCESSLOG_STREAM_SUFFIX = "access-log";
   public static final String CHANGELOG_REPLICATION_FACTOR = STORE_PREFIX + "%s.changelog.replication.factor";
   public static final String CHANGELOG_MAX_MSG_SIZE_BYTES = STORE_PREFIX + "%s.changelog.max.message.size.bytes";
-  public static final int DEFAULT_CHANGELOG_MAX_MSG_SIZE_BYTES = 1000000; // slightly less than 1 MB
+  public static final int DEFAULT_CHANGELOG_MAX_MSG_SIZE_BYTES = 1048576; // slightly less than 1 MB
```

Comment:
Comment is no longer consistent with value

Reply:
Removed the comment.